modules/nlu/src/backend/engine.ts

```diff
@@ -192,4 +193,8 @@ export default class Engine implements NLUEngine {
     // error handled a level highr
     return Predict(input, Engine.tools, this.predictorsByLang)
   }
+
+  async elect(predictionOutput: PredictOutput): Promise<PredictOutput> {
```

Comment:
do you really need to add this as a method ? I mean especially that this is a legacy refactor, we don't want to add code